util/misc_test.go

```diff
@@ -152,9 +155,12 @@ func (s *testMiscSuite) TestBasicFunc(c *C) {
 		Time:    time.Now(),
 		State:   1,
 		Info:    "test",
+		StmtCtx: &stmtctx.StatementContext{
+			MemTracker: memory.NewTracker(stringutil.StringerStr(""), -1),
+		},
 	}
-	row := pi.ToRow(false)
-	row2 := pi.ToRow(true)
+	row := pi.ToRowForShow(false)
```

Comment:
Can we rewrite this test to not call `pi.ToRowForShow(false)`? Such that we can remove the parameter `full` in `ToRowForShow(full bool)`

Reply:
`SHOW FULL PROCESSLIST` calls `pi.ToRowForShow(true)` and `SHOW PROCESSLIST` calls `pi.ToRowForShow(false)`.